### PR TITLE
Clean Code ToDo application

### PR DESCRIPTION
**Pull request by TC-rGalvan**
This PR includes refactoring for todo list program and user-friendly interaction

### CODE REVIEW

**Comment by TC-rGalvan** [comment]
It needs to merge into my own repository, not into this sample repo.

File: csharp/Clean Code/ToDo/Program.cs [hunk 1]
```diff
@@ -5,106 +5,131 @@ namespace ToDo
 {
     internal class Program
     {
-        public static List<string> TL { get; set; }
+        public static int option {get;set;}
+        public static List<string> TaskList { get; set; }
```
Comment (TC-oRovira) [inline csharp/Clean Code/ToDo/Program.cs:9]: I think that the naming convention for properties is PascalCase, so Options will be better also to maintain consistency between your 2 properties

**Review by TC-oRovira — COMMENTED** [review]
Everything else looks good to me

File: csharp/Clean Code/ToDo/Program.cs (hunk 1 above)
Comment (TC-rGalvan) [reply csharp/Clean Code/ToDo/Program.cs:9]: I appreciate you noticing. I have fixed it and I'll push it along with the unit tests.

**Pull request closed without merging**